java-util/src/main/java/io/druid/java/util/common/io/smoosh/SmooshedFileMapper.java

```diff
@@ -97,7 +97,7 @@ public static SmooshedFileMapper load(File baseDir) throws IOException
   private final Map<String, Metadata> internalFiles;
   private final List<MappedByteBuffer> buffersList = Lists.newArrayList();
 
-  private SmooshedFileMapper(
+  public SmooshedFileMapper(
```

Comment:
I want to use V9IndexLoader.load(SmooshedFileMapper smooshedFiles, File inDir, ObjectMapper mapper), this method requires SmooshedFileMapper.
It's fine SmooshedFileMapper can change behaviors at any time, I extend SmooshedFileMapper in my extension and will have unit test to ensure everything is working properly.